pkg/apis/kf/v1alpha1/space_defaults.go

```diff
@@ -21,6 +21,9 @@ import (
 const (
 	// DefaultBuilderImage contains the default buildpack builder image.
 	DefaultBuilderImage = "gcr.io/kf-releases/buildpack-builder:latest"
+
+	// DefaultDomain contains the default domain.
+	DefaultDomain = "example.com"
```

Comment:
WDYT about `<ns>.kf.cluster.local` instead so a misconfiguration doesn't accidentally hit the real `example.com`?